tests/landIce/FO_GIS/input_fo_humboldt_frosch_fluxdiv.yaml

```diff
@@ -327,7 +327,7 @@ ANONYMOUS:
                     Adding Layers Strategy: CrsGraph
                     Solver:                                                 # Solver on the first level
                       SolverType: Amesos2                                   # Solver package: Amesos2 or Ifpack2
-                      Solver: Klu                                           # Solver name (depends on the solver package): Klu, RILUK, ...
+                      Solver: ${DIRECT_SOLVER}                              # Solver name (depends on the solver package): Klu, RILUK, ...
```

Comment:
Will the final comparison be the same regardless of the solver?

Reply:
Close enough that the test passed.. After all they are direct solvers.

Reply:
Okay great